scripts/gha/ui_testing/uitest_apple/FirebaseCppUITestApp/AppDelegate.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AppDelegate.swift
+//  FirebaseCppUITestApp
+//
+//  Created by Mou Sun on 3/19/22.
+//  Copyright © 2022 Google. All rights reserved.
+//
+
```

Comment:
This should have our standard copyright notice.  See an [example swift file in the iOS Github Repo](https://github.com/firebase/firebase-ios-sdk/blob/8f5e7f0b376279f40ded653e04730fa2b23e0a89/FirebaseMessaging/Apps/SwiftUISample/SwiftUISample/ContentView.swift).